sources/packages/web/src/types/contracts/institution/EducationProgramOfferingDto.ts

```diff
@@ -20,14 +20,12 @@ export interface EducationProgramOfferingDto {
  */
 export enum OfferingStatus {
   Approved = "Approved",
-  // This status will be changed to CreationPending.
-  Pending = "Pending",
-  // This status will be changed to CreationDeclined.
-  Declined = "Declined",
-  // This status will be changed to ChangeUnderReview.
-  UnderReview = "Under review",
-  // This status will be changed to ChangeAwaitingApproval.
-  AwaitingApproval = "Awaiting approval",
+  // Status with respect to offering creation.
```

Comment:
Add a comment on the API side to also extend these to add some business comments. Please ensure that this one is also updated.